src/ClimaAtmos/Parameterizations/TurbulenceConvection/src/Grids/Grids.jl

```diff
@@ -0,0 +1,69 @@
+module Grids
+
+import Base
+export Grid, over_elems, over_elems_real, over_elems_ghost
+export first_cell_above_surface, get_z
+
+struct Grid{T}
+  n_elem :: Int
+  dz :: T
+  dzi :: T
+  dzi2 :: T
+  z :: Vector{T}
+  n_ghost :: Int
+  zc_top :: T
+  zn_top :: T
+  zc_surf :: T
+  zn_surf :: T
+end
+
+function Grid(z_min::T, z_max::T, n_elem_real::Int, n_ghost::Int = 1) where T
+  n_elem = n_elem_real+2*n_ghost
+  dz = T((z_max-z_min)/T(n_elem_real))
```

Comment:
`T`'s are unnecessary here
```suggestion
  dz = (z_max-z_min)/n_elem_real
```